compiler-rt/lib/radsan/radsan.h

```diff
@@ -8,17 +8,31 @@
 
 #pragma once
 
-#define RADSAN_EXPORT __attribute__((visibility("default")))
+#include "sanitizer_common/sanitizer_atomic.h"
+#include "sanitizer_common/sanitizer_internal_defs.h"
+#include "radsan_flags.h"
 
-extern "C" {
+namespace radsan {
```

Comment:
It would be nice to be able to keep this `radsan.h` header file to just the public-facing API methods. I wonder if we could move these functions to another header/impl combination like `radsan_globals.h` and `radsan_globals.cpp`?